### ISSUE TITLE
How do you reset ParamStore when clicking on a <Link /> element?

### ISSUE THREAD

**Issue by dvmorris** [poetic/param-store#3]
I am trying to implement navigation in my app using ParamStore, and my query string is not getting reset when I navigate from a detail page (like an edit form page), to another page that is not object-specific:

```
// navigate to "edit" form
<Link params={{  path: 'edit-group', groupId: this.props['group']._id  }}>Edit</Link> // puts groupId in query string
// navigate to another page using nav menu
<Link params={{  path: 'support'}}>Support</Link> // does not clear groupId from query string
// navigate to create form and redirect after successful form submit
ParamStore.reset({ path: 'groups' }); // this clears the "groupId" param from the query string as expected
```

Any ideas?

**Comment by Chun-Yang** [comment]
@dvmorris This is not doable right now. The only way is to change it to a normal html tag and attach 'onClick' event and use 'ParamStore.reset' function.

**Comment by Chun-Yang** [comment]
@dvmorris We published a new version 1.0.0 .
1. reset changed to setAll
2. Link use setAll under the hood.

So now Link only setAll instead of set.
